components/filter/test/filter.test.js

```diff
@@ -495,10 +495,11 @@ describe('d2l-filter', () => {
 				expect(dimensions[0].dimensionKey).to.equal('dim');
 				expect(dimensions[0].cleared).to.be.false;
 				expect(dimensions[0].changes.length).to.equal(2);
-				expect(dimensions[0].changes[0].valueKey).to.equal('2');
-				expect(dimensions[0].changes[0].selected).to.be.true;
-				expect(dimensions[0].changes[1].valueKey).to.equal('1');
-				expect(dimensions[0].changes[1].selected).to.be.false;
```

Comment:
@svanherk this was the Safari flake I was chatting with you about. Because it's the underlying list that un-selects the one item when a new one is selected, I don't know that there's any guarantee the changes will be in a particular order. So I updated this to not care about the order.